api-changelog.md

```diff
@@ -1,3 +1,8 @@
+# Version 1.1.0 (2024-05-30)
+
+## Added
+- AI Assistant: API endpoints to start a conversation with the AI assistant, ask a new question, and list the conversations and messages.
```

Comment:
Reword to "API endpoints to start a conversation with the AI assistant, ask questions, and list conversations and messages."

Reply:
done